### DISCUSSION TITLE
Improved REST API docs 🎉

### DISCUSSION THREAD

**Discussion by JanCizmar** [tolgee/tolgee-platform#2235] [Announcements]
Hey! 👋

We've just updated the [API docs](https://tolgee.io/api), so
- They contain not only the endpoints accessible with [Project API key](https://tolgee.io/platform/account_settings/api_keys_and_pat_tokens#project-api-keys), but also with [Personal access tokens](https://tolgee.io/platform/account_settings/api_keys_and_pat_tokens#personal-access-tokens-pat-tokens) and JWT tokens we use for the UI. 🔑
- The authentication methods are now documented in the right OpenAPI way 📄
- You can get generated code for your favorite programming language and HTTP library ☕️ 
- They have a fresh new design 🍏
- They are now generated as static files when a new Tolgee Platform release occurs. This improves the SEO, and also, you can search in them 🔎

Any feedback is welcomed! 🙏

**Comment by vetrivelavan** [comment]
is it possible to sort the keys in ascending or descending in a specific project?

**Reply by JanCizmar** [reply]
Hello, It's not. What is your use case for such feature?

**Reply by HZZformGD** [reply]
@JanCizmar  It's about some statistical requirements from the project manager. In my project, I need to track which new entries have been added this month or recently. Currently, we manually track this using tags, but since many engineers are involved, some may forget or be reluctant to add them. I hope this feature can be added. The current sorting method does not meet my needs.

**Reply by JanCizmar** [reply]
I am forwarding this to our UX designer:
@ZuzanaOdstrcilova Can you please consider also sorting by date added?

**Reply by HZZformGD** [reply]
> I am forwarding this to our UX designer: @ZuzanaOdstrcilova Can you please consider also sorting by date added?

Thank you for @JanCizmar your response .  Before UX agrees, I want to ask if the API (this one: https://app.tolgee.io/v2/projects/:projectId/translations/:languages) already supports sorting by the order of addition?

**Reply by JanCizmar** [reply]
Hello! This one definitely not, but this one will support it in the future.

https://docs.tolgee.io/api/get-translations
